### ISSUE TITLE
Set up online Demo

### ISSUE THREAD

**Issue by davidak** [MycroftAI/mimic#88]
The best way to present this software is to have a live demo on the website where you can write text, select a voice and click "speak".

See this example: http://tts.speech.cs.cmu.edu:8083/

**Comment by forslund** [comment]
Good idea, I'll look into getting a server/host to run it. Do you have any suggestions on implementation?

**Comment by aatchison** [comment]
I can get you server time, etc.

**Comment by forslund** [comment]
@aatchison, thanks I'll reach out to you on slack when I get some spare time (likely monday) for the specifics.

**Comment by aatchison** [comment]
cool cool

**Comment by davidak** [comment]
@forslund a really simple implementation could look like this. if you have enough resources to handle the requests, the startpage would be a good place.

![bildschirmfoto 2016-10-28 um 19 33 38](https://cloud.githubusercontent.com/assets/91113/19816001/bd1cf48e-9d45-11e6-90c2-1aaa3b220426.png)

or you can create a "Demo" page linked on the menu to the left.

The [Flite Demo](http://tts.speech.cs.cmu.edu:8083/) uses [GoFlite](https://github.com/happyalu/goflite) on the server-side. Since Mimic is based on Flite, it could be easy to implement.
